### DISCUSSION TITLE
Integrate Meilisearch Cloud with GCP Marketplace

### DISCUSSION THREAD

**Discussion by ahmednfwela** [meilisearch/product#668] [Feedback & Feature Proposal]
MongoDb Atlas has an interesting feature, where you can use your GCP balance as a MongoDb balance, documented [here](https://www.mongodb.com/docs/atlas/billing/gcp-self-serve-marketplace/) and uses this [Marketplace Product](https://console.cloud.google.com/marketplace/product/mongodb/mdb-atlas-self-service)

It would be great if Meilisearch Cloud had a similar billing experience (note that the instance hosting itself doesn't have to be on GCP)

**Comment by davelarkan** [comment]
Hi @ahmednfwela ,

Thanks for the feedback. We're definitely open to alternate billing options.

I'll explore the details of how this works with MongoDb and what would be required to do something similar with Meilisearch Cloud.

I wonder could you provide some more context around why this would be a preferred billing experience for you?

**Reply by ahmednfwela** [reply]
as a business, the ability to pay everything in one place is very convenient to have ! (unified reports , unified cost breakdown , unified balance ...)
